client/src/AddFriend.js

```diff
@@ -1,19 +1,20 @@
 import React from 'react'
 import axios from 'axios'
+import { Button, Spinner} from 'react-bootstrap'
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
[nit] I'm pretty sure you don't need to import CSS files to add class names -- they should kind of just work?

Reply:
The spinners dont seem to work without it :c